### DISCUSSION TITLE
寻求扶正的可能性？

### DISCUSSION THREAD

**Discussion by tanukihee** [whutug/whu-thesis#316] [General]
虽然 WHU 从来没有给出过官方的 LaTeX 模板，但部分学院（比如物院）是推荐使用 LaTeX 写毕业论文的（却让学生自己网上找模板）

也许 WHUTUG 可以寻求与武大本科生院/研究生院合作，使该模板得到官方承认，俗称「扶正」？

**Comment by tanukihee** [comment]
[干货！毕业论文设计详细技巧都在这里...](https://mp.weixin.qq.com/s?__biz=MzUyNjAzMjAwMg==&mid=2247511462&idx=1&sn=694643fd3f40f1bf48cb7b25a402c519)

![image](https://user-images.githubusercontent.com/34062491/115968456-a9463880-a56a-11eb-8f10-f546223c1ef8.png)

**Comment by sikouhjw** [comment]
扶正就得考虑谁去长期维护的问题了（

**Comment by imfing** [comment]
长期维护的话，稳定版能够满足使用者的需求，并且格式正确能够被学校老师接受，长期维护的成本不会非常高。LaTeX 因为有学习成本，只能作为 Word 之外的一个选项。不过要是有更多的人用的话，的确可以和老师接触，得到官方认可。
我推荐联系数院的 [黄正华](http://aff.whu.edu.cn/huangzh/) 老师，最初他自己做了几份 LaTeX 的模板，我觉得他肯定会支持这个项目的。

**Comment by sikouhjw** [comment]
> 长期维护的话，稳定版能够满足使用者的需求，并且格式正确能够被学校老师接受，长期维护的成本不会非常高。LaTeX 因为有学习成本，只能作为 Word 之外的一个选项。不过要是有更多的人用的话，的确可以和老师接触，得到官方认可。
> 我推荐联系数院的 [黄正华](http://aff.whu.edu.cn/huangzh/) 老师，最初他自己做了几份 LaTeX 的模板，我觉得他肯定会支持这个项目的。

稳定就是最大的问题。运动是绝对的，需求是无限的。总会出现有个版本的 texlive 无法实现的需求，固定版本死路一条（
而且格式「正确」这种模糊的说法，我在 ThesisUESTC 那边看到过各种「某些人通过、某些人不通过」的情况，除非有人愿意担保/出个文件证明 LaTeX 排出来的是合格的（然而大家都懂的原因，LaTeX 很难做到跟 word 一样，或者由于使用者的水平问题，排出来是一堆垃圾），否则感觉扶正很难

**Comment by tanukihee** [comment]
whu 似乎没有一个类似 tuna 的校级组织/协会能够挂靠（不然也不会有 whutug 这种「野鸡组织」了😅），如果能挂一个正规协会的羊头，「官方性」也能强上许多

长期维护确实是问题。看来比起扶正，多找几个维护者最要紧😅

**Comment by zepinglee** [comment]
> whu 似乎没有一个类似 tuna 的校级组织/协会能够挂靠（不然也不会有 whutug 这种「野鸡组织」了😅），如果能挂一个正规协会的羊头，「官方性」也能强上许多

感觉不是很重要。主要是研究生院有老师愿意沟通。

**Comment by imfing** [comment]
> whu 似乎没有一个类似 tuna 的校级组织/协会能够挂靠（不然也不会有 whutug 这种「野鸡组织」了😅），如果能挂一个正规协会的羊头，「官方性」也能强上许多

计算机学院好像应该没有类似的组织/协会（祖上太穷），我以前是微软俱乐部的主席，不知道现在里面会不会有人愿意维护一个 LaTeX 的项目。
我觉得还是得找到维护者，再不济比较稳定了每年更新一下 README 🤣
个人认为学校层面因为已经有了一劳永逸的 Word 版本，所以推行一个 LaTeX 版本会比较困难，不过可以试着和学校沟通

**Comment by sikouhjw** [comment]
提供个思路：我的导师打算找研究生院沟通 `gdutthesis` 的扶正问题，应该是挂到官网背书 + 不断根据需求更新迭代，以项目的形式委托给我（们）
